src/mods.ts

```diff
@@ -1,15 +1,10 @@
-import core from 'src/core'
-import { addSuffix, removeSuffix, updateValues } from 'src/utils'
-import { targets } from 'src/index'
+import core from './core'
+import { addSuffix, removeSuffix, updateValues } from './utils'
+import { targets } from './index'
 
 // Shared modifications
 const updatedBoxShadow = updateValues(core.BoxShadow, '')
-const updatedEasing = updateValues(core.Easing, {
-  x1: 0.475,
-  y1: 0.425,
-  x2: 0,
-  y2: 0.995,
-})
+const updatedEasing = updateValues(core.Easing, 'cubic-bezier(0.475, 0.425, 0, 0.995)')
```

Comment:
Is this different then the code on the left? I didn't write it, so I'm actually not sure what it is doing.

Reply:
TBH, I'm also not sure what the purpose of this is, because the values provided are identical to the default values.  This was the only way I could get it to work. Out of all the changes I made this is the only one I'm not 100% certain about.

Reply:
If we notice any issues after we publish we will have to figure out how to fix it.